src/django_linear_migrations/apps.py

```diff
@@ -106,6 +107,13 @@ def names(self) -> set[str]:
             if not is_pkg and name[0] not in "_~"
         }
 
+    @cached_property
+    def plan(self) -> list[tuple[str, str]]:
```

Comment:
for every MigrationDetails it would rebuild graph, but making one global function to get graph plan is hard because it would try to import module like in MigrationDetails.__init__